pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/SegmentCreationJob.java

```diff
@@ -69,6 +75,9 @@ public SegmentCreationJob(String jobName, Properties properties) throws Exceptio
     _outputDir = getOutputDir();
     _stagingDir = new File(_outputDir, TEMP).getAbsolutePath();
     _depsJarPath = _properties.getProperty(PATH_TO_DEPS_JAR, null);
+    _hosts = _properties.getProperty("push.to.hosts").split(",");
```

Comment:
I think that you need to provide the backward compatibility in case when no push location is provided. This will affect to people outside who are using this push job.

Reply:
good call, fixed